helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -25,9 +25,6 @@ data:
         address: {{ .Values.alertmanager.address }}
         baseDomain: {{ .Values.alertmanager.host }}
         logLevel: {{ .Values.alertmanager.logLevel }}
-        storage:
-          createPVC: {{ and .Values.alertmanager.storage.enabled .Values.alertmanager.storage.createPVC }}
-          size: {{ .Values.alertmanager.storage.pvcSize }}
```

Comment:
are you sure you want to remove size field ?

Reply:
Actually no I'm adding it back